.env

```diff
@@ -32,3 +32,9 @@ ES_ENABLED_DOCKER=false
 #uncomment to enable Idam Stub
 #IDAM_STUB_SERVICE_NAME=http://ccd-test-stubs-service:5555
 #IDAM_STUB_LOCALHOST=http://localhost:5555
+
+#XUI WEBAPP Oauth2 secrets
+BEFTA_S2S_CLIENT_SECRET_OF_XUI_WEBAPP=OOOOOOOOOOOOOOOO
+
+#ACA Oauth2 secrets
+BEFTA_S2S_CLIENT_SECRET_OF_AAC_MANAGE_CASE_ASSIGNMENT=AAAAAAAAAAAAAAAA
```

Comment:
eol at eof pls

Reply:
Fixed in https://github.com/hmcts/ccd-docker/pull/103/commits/2a11ab59294f83f9ce45b5e51cda3ea2bc309264